tests/integration.rs

```diff
@@ -1382,10 +1384,13 @@ fn test_vhost_user_blk(
 
             // Add RAM to the VM
             let desired_ram = 1024 << 20;
-            resize_command(&api_socket, None, Some(desired_ram), None, None);
-
-            thread::sleep(std::time::Duration::new(10, 0));
```

Comment:
You simply started the VM with an even-monitor option, but you never actual check the output from event-monitor. I don't understand how you can avoid the timeout in this way. It does not look right to me.

Reply:
Also, you will need to check if the action (say memory hotplug) is actually **completed** when the corresponding event is reported to the "event-monitor" file. This is why we need a hard-coded `sleep` in the first place.

Reply:
> You simply started the VM with an even-monitor option, but you never actual check the output from event-monitor. I don't understand how you can avoid the timeout in this way. It does not look right to me.

I thought this at first but he does then pass it into the `resize_command()` function which uses it.

Reply:
> Also, you will need to check if the action (say memory hotplug) is actually **completed** when the corresponding event is reported to the "event-monitor" file. This is why we need a hard-coded `sleep` in the first place.

We might be a able to add an event when the guest acknowledges the ACPI notification - but that might only be a available for cpu resize...

Reply:
As @rbradford mentioned events_file is passed to resize_command which checks for "resizing" and "resized" events.

Reply:
But it does not check for the "completion" of the resize. What's being checked from `resize_command` is simply the start of the resize.

Reply:
@likebreath how can I check for the completion of the resize?
isn't "resized" event doing that?

This is the last event in the events file

{
  "timestamp": {
    "secs": 25,
    "nanos": 717714703
  },
  "source": "vm",
  "event": "resized",
  "properties": null
}

Reply:
If you check the source code you can see exactly when `resized` is fired; it's just after the guest notification. 

If you read the ACPI spec/Linux kernel to see if it calls any AML method when the resize is done - maybe `_OST` and then you could fire a new event e.g. `guest_resized` to handle that.